collector/collector_test.go

```diff
@@ -367,38 +369,10 @@ func TestCollectMetrics(t *testing.T) {
 				})
 			})
 		})
```

Comment:
Would you like to add tests covered cases for filesystem metrics and alsonew ones (hugetlb, blkio and cpuset statistics)?